go/proxyd/metrics.go

```diff
@@ -20,6 +21,8 @@ const (
 	MethodUnknown = "unknown"
 )
 
+var PayloadSizeBuckets = []float64{10, 50, 100, 500, 1000, 5000, 10000, 100000, 1000000}
```

Comment:
@mslipper these buckets should suffice. lmkwyt

Reply:
Yea they LGTM